File: Programming Basics with JS/03. Conditional Statements Advanced - Lab/skiTrip.js
function solve(arg, arg2, arg3,) {
    let days = Number(arg);
    let type = String(arg2);
    let opinion = String(arg3);
    let nights = days -1;
    let result = 0;

    switch (type){
        case "room for one person":
            result = nights * 18;
            break;
        case "apartment":
            result = nights * 25;
            if(days<10){
                result *= 0.7;
            } else if(days>10 && days<15){
                result *= 0.65;
            } else {
                result *= 0.5;
            }
            break;
        case "president apartment":
            result = nights * 35;
            if(days<10){
                result *= 0.9;
            } else if (days>=10 && days<=15){
                result *= 0.85;
            } else{
                result *= 0.8;
            }
            break;
    }

    if(opinion === "positive"){
        result *= 1.25;
    } else{
        result *= 0.9;
    }
    
    console.log(result.toFixed(2));

}

solve("30",
"president apartment",
"negative")

File: Programming Basics with Java/03. Conditional Statements Advanced - Lab/src/SkiTrip.java
import java.util.Scanner;

public class SkiTrip {
    public static void main(String[] args) {
        Scanner scan = new Scanner (System.in);

        int days = Integer.parseInt(scan.nextLine()) - 1;
        String roomType = scan.nextLine();
        String grade = scan.nextLine();
        double price = 0;

        if(roomType.equals("room for one person")){
            price = days * 18;
        } else if(roomType.equals("apartment")){
            price = days * 25;

            if(days < 10){
                price *= 0.7;
            } else if(days>= 10 && days<= 15){
                price *= 0.65;
            } else {
                price *= 0.5;
            }
        } else if (roomType.equals("president apartment")) {
            price = days * 35;

            if(days < 10){
                price *= 0.9;
            } else if(days>= 10 && days<= 15){
                price *= 0.85;
            } else {
                price *= 0.80;
            }
        }

        if(grade.equals("positive")){
            price *= 1.25;
        } else {
            price *= 0.9;
        }

        System.out.printf("%.2f", price);
    }
}
